src/init-agent-script.ts

```
@@ -363,1219 +144,1303 @@ const readWideString = (address: any) => {
+/*---------------------Hook---------------------*/
+/*
+接收消息回调 3.9.10.27
+*/
+const recvMsgNativeCallback = (() => {
+
+  const nativeCallback = new NativeCallback(() => { }, 'void', ['int32', 'pointer', 'pointer', 'pointer', 'pointer', 'int32'])
+  const nativeativeFunction = new NativeFunction(nativeCallback, 'void', ['int32', 'pointer', 'pointer', 'pointer', 'pointer', 'int32'])
+
+  try {
+      Interceptor.attach(
+          moduleBaseAddress.add(offsets.kDoAddMsg), {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)